### PR TITLE
Merge to Ethan dev branch

### PR DESCRIPTION
**Pull request by EthanCornell**
Merge to Ethan dev branch

### CODE REVIEW

File: sys/netgraph/ng_base.c [hunk 1]
```diff
@@ -73,6 +73,8 @@
 #include <netgraph/netgraph.h>
 #include <netgraph/ng_parse.h>
 
+#include <sys/epoch.h> // Include the epoch header
```
Comment (bsdimp) [inline sys/netgraph/ng_base.c:76]: Remove useless comment

**Comment by bsdimp** [comment]
Please use C comments.
Please don't just comment out code.

**Comment by bsdimp** [comment]
@glebius can you review this?

**Comment by bsdimp** [comment]
Ethan, thanks for submitting this. I did a quick style check. But i asked someone to review it that knows epoch.

**Comment by concussious** [comment]
Commented 2 days ago *via email*?
Please excuse off topic, but please, how?
Searching "github via email" was unhelpful.

**Comment by bsdimp** [comment]
Is this ready? Or still stress testing?

**Comment by ehem** [comment]
> Commented 2 days ago _via email_? Please excuse off topic, but please, how? Searching "github via email" was unhelpful.

If you're subscribed to a given message stream, GitHub will send e-mail when comments are made.  You can reply to that e-mail to have your response appear.

> Is this ready? Or still stress testing?

This looks like someone experimenting with git/GitHub who accidentally selected the wrong spot.  Appears @EthanCornell likely created this pull request by mistake.  Given the comment [above](https://github.com/freebsd/freebsd-src/pull/1291#issuecomment-2167029005) I would tend to close this as the last comment was 2 weeks ago and there hasn't been any activity by @EthanCornell

**Comment by concussious** [comment]
Thanks @ehem! I've been wondering this for a long time.

**Comment by bsdimp** [comment]
Closing... If the stress tests reveal anything and you want to pursue this or other changes, create a new pull request. thanks!

**Pull request closed without merging**